frontend/src/utils/utils.ts

```diff
@@ -158,6 +158,15 @@ export const formatDateTime = (date: string | undefined) => {
   return !!date ? moment(date).format('YYYY-MM-DD hh:mm a') : '';
 };
 
+export const formatApiDateTime = (date: string | undefined) => {
```

Comment:
Add some jsdoc and it'll be even clearer what the function does with intellisense 👍